test/runner/utils.ts

```diff
@@ -1,5 +1,9 @@
 import { readFile, statSync, readdirSync, writeFile, existsSync } from 'fs';
 
+const PrettyError = require('pretty-error');
```

Comment:
I guess you copied it from somewhere, but do not use `require()` here for consistentcy. :)

Reply:
I had to use 'require' because there are no ts types for 'pretty-error'

Reply:
declare module 'pretty-error';
```

Reply:
Can you take over this PR and merge it?